Blockzilla/URLBar.swift

```diff
@@ -108,7 +108,7 @@ class URLBar: UIView {
     private var hideToolsetConstraints = [Constraint]()
     private var showToolsetConstraints = [Constraint]()
 
-    private var compressBar = false {
+    private var compressBar = true {
```

Comment:
Can you explain why this is the fix?

Reply:
Oh yes!

Originally it was impossible to get to the browser view without going through the flow of searching first.

When you enter the browser through an external link the state would get set to `browsing`:
https://github.com/mozilla-mobile/focus-ios/blob/main/Blockzilla/URLBar.swift#L650

and set compressBar to false. But since we have this guard here: https://github.com/mozilla-mobile/focus-ios/blob/main/Blockzilla/URLBar.swift#L113 it wont get the constraints set properly.

The default state is true https://github.com/mozilla-mobile/focus-ios/blob/main/Blockzilla/URLBar.swift#L639 which is what all of the original constraints are setup for so those this shouldn't break anything.